haystack_experimental/dataclasses/tool.py

```diff
@@ -237,3 +327,41 @@ def deserialize_tools_inplace(data: Dict[str, Any], key: str = "tools"):
             deserialized_tools.append(Tool.from_dict(tool))
 
         data[key] = deserialized_tools
+
+
+def _standardize_tool_definition(llm_specific_tool_def: Dict[str, Any]) -> Optional[Dict[str, Any]]:
```

Comment:
This utility function only ensures compatibility between Haystack and the new library. I don't expect to use it elsewhere in Haystack if we do things right.
Could it be moved to the library?

Reply:
Good point. I'll move it.